.circleci/config.yml

```diff
@@ -13,62 +13,15 @@ orbs:
   slack: circleci/slack@4.9.3
   codecov: codecov/codecov@3.2.3
 
+parameters:
+  run_coverage:
+    type: boolean
+    default: false
+
 commands:
-  install-postgres:
-    description: Install PostgreSQL
```

Comment:
What happens to all these external requirements? I can't find Postgres in `ci-requirements.txt`.

Reply:
it's in the base image. This "command" was defined but wasn't actually being used anyway